index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/34795ce01c.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav>
+        <header class="logo"><a href="#" target="_blank">My Logo</a></header>
+        <i class="fa-solid fa-bars"></i>
+    </nav>
+    <section class="headline">
+        <img src="header-shapes mobile.png" alt="shapes">
```

Comment:
- [ ] Please kindly use CSS background-image property to add this image to your project. If an image is not part of the content (aesthetic reasons) it is better to use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. ℹ️

Reply:
@OyePriscilla Issue is resolved. I updated the .html file and used CSS background-image property.